Tests/Mocks/MockStorageContainer.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  MockStorageContainer.swift
+//  SessionTools-iOSTests
+//
+//  Copyright © 2018 Bottle Rocket Studios. All rights reserved.
+//
+
+import Foundation
+import SessionTools
+
+/// An in memory container for testing.
+public class MockStorageContainer<T: Codable>: SessionContainer {
+    
+    private var localStorage: [String: Data]
+    
+    public init() {
+        localStorage = [:]
```

Comment:
Any reason for assigning the initial value of `localStorage` here instead of just assigning it in the property declaration above?

Reply:
I was thinking in the future we might want to pass in data for testing, but nothing uses that now, so I've move it up to the property declaration.